### PR TITLE
[WIP] ONNX _contrib_interleaved_matmul_selfatt_valatt and LayerNorm

### PR DESCRIPTION
**Pull request by waytrue17**
## Description ##
Support `_contrib_interleaved_matmul_selfatt_valatt` to ONNX, rewrite `LayrNorm` conversion. 
Add tests for the ops

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### CODE REVIEW

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 1]
```diff
@@ -1656,16 +1657,15 @@ def convert_slice_axis(node, **kwargs):
     if not ends or ends == 'None':
         # ONNX doesn't support None for ends. Since ends=None depicts
         # length of dimension, passing dimension in this case.
-        in_shape = kwargs['in_shape'][0]
-        ends = in_shape[axes]
+        ends = sys.maxsize
```
Comment (Zha0q1) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:1660]: I rewrote conversion logic for slice_axis with the new onnx slice so you might need to rebase to my pr.
https://github.com/apache/incubator-mxnet/pull/19677

File: tests/python-pytest/onnx/test_operators.py [hunk 2]
```diff
@@ -98,11 +98,14 @@ def test_onnx_export_arange_like(tmp_path, dtype):
 
 
 def test_onnx_export_layernorm(tmp_path):
-    M = def_model('LayerNorm', axis=1)
-    x = mx.nd.array([[1,3],[2,4]], dtype='float32')
-    gamma = mx.random.uniform(0, 1, x[0].shape, dtype='float32')
-    beta = mx.random.uniform(0, 1, x[0].shape, dtype='float32')
-    op_export_test('LayerNorm', M, [x, gamma, beta], tmp_path)
+    x = mx.nd.random.uniform(1, 2, (3, 4, 5), dtype=dtype)
```
Comment (Zha0q1) [inline tests/python-pytest/onnx/test_operators.py:101]: I think dtype is not defined?

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py (hunk 1 above)
Comment (waytrue17) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: I reverted this change so we can have it fixed from you PR

File: tests/python-pytest/onnx/test_operators.py (hunk 2 above)
Comment (waytrue17) [reply tests/python-pytest/onnx/test_operators.py:101]: Added dtype, thanks for pointing it out

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 3]
```diff
@@ -53,6 +53,7 @@
 """
 
 import re
+import sys
```
Comment (Zha0q1) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:56]: nit: we can probably remove this line now

**Review by Zha0q1 — APPROVED** [review]
LGTM

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py (hunk 3 above)
Comment (waytrue17) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: Removed it, thanks!

**Pull request merged**